### ISSUE TITLE
`iscan` documentation is confusing / ambiguous

### ISSUE THREAD

**Issue by mstfdkmn** [irods/irods#6369]
- [x] main
- [x] 4-3-stable

---

### iRODS Version, OS and Version
4.2.10 Centos7

### What did you try to do?
`iscan /local/to/test.txt` is run to know whether this object is registered in iRODS or not.

### Expected behavior
Depending on if the local file content is registered in iRODS, some informative messages. 

### Observed behavior (including steps to reproduce, if applicable)
```
-bash-4.2$ iscan /local/to/test.txt
/local/to/test.txt is not registered in iRODS
```
and `ils` shows:
```
/u0137480/home/rods:
  test.txt
```
However `iscan -d /u0137480/home/rods/test.txt` seems working normal.
Also, `iscan -h` could have included a usage example. Thanks.

**Comment by mstfdkmn** [comment]
I am confirming that:

1. Yes, the path exists on the host where `iscan `is being run.
2. Yes, that is what I see as an output of the `ils -L` and also I checked and I saw that it is located in the storage server.
3. No, the physical path for the replica is /storageServerName//u0137480/home/rods/test.txt.

I tested again inside a test container in which is also a storage(resource) to an existing iRODS server and I see that the `iscan` command can only work for:

1. checking if the physical files corresponding to iRODS objects exists on the resource servers. - an admin can `iscan -d test.txt`
2. checking a physical path to know if a corresponding logical object(path) is registered in irods. - `iscan /vault/home/rods/test.txt`

Then it seems the documentation below a bit confusing -a at least for me! How I am understanding the document is like 'a local file and/or a directory is an object and path on where I upload it to irods'. Could `iscan - h` perhaps include an usage example later? Thank you.

> If the input is a local data file or a local directory, iscan
> checks if the content is registered in iRODS.
> 
> 
> Full path must be used for local files and directories.
